### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by lvh**
cc @rm-hull for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,20 @@ In order to release to Clojars, you'll need to set `CLOJARS_USERNAME` and `CLOJA
 e.g. `CLOJARS_USERNAME=lshift CLOJARS_PASSWORD=<LShift Clojars password> lein release` in each of the cloverage and lein-cloverage folders.
 
 ## Changelog
+
+1.0.7 (WIP)
+- Features
+  - Add codecov.io support with the `--codecov` flag (*78)
```
Comment (rm-hull) [inline README.md]: Should be #78 rather than *78 ?

File: README.md [hunk 2]
```diff
@@ -60,6 +60,20 @@ In order to release to Clojars, you'll need to set `CLOJARS_USERNAME` and `CLOJA
...
+  - Add lcov (e.g. coverlay) support with the `--lcov` flag (#114)
+- Improvements
+  - Coverage fn (internal hot loop) optimization (#90)
+  - Dependency upgrades, including running tests on OpenJDK 8 (#105)
```
Comment (rm-hull) [inline README.md]: OracleJDK 8

**Comment by lvh** [comment]
Thanks! I've addressed your comments.

**Comment by lvh** [comment]
I guess this is good to go :)

**Pull request merged**